app/models/split.rb

```diff
@@ -82,8 +119,34 @@ def create_decision!(params = {})
     build_decision(params).tap(&:save!)
   end
 
+  def registry
+    if try(:feature_incomplete?)
+      knock_out_weightings(super)
+    else
+      super
+    end
+  end
+
   private
 
+  def knock_out_weightings(registry_hash, to: "false")
```

Comment:
What is the plan for being able to go with something other than `'false'`

Reply:
Remote kills will support arbitrary splits because it’s basically an “oh god; this version range has a catastrophic bug in it - what can i grab onto to mitigate?” Tool, so you might e.g. go to 100% treatment, 100% control, 100% whatevs. But feature incomplete is for conventional feature gates with true/false variants.

Reply:
(note that you can use a remote kill instead if you accidentally screwed up the variant names on a feature gate - it's just a more powerful version of the tool that requires more configuration because it needs a first version and identity (a reason slug) for future mutability)